.travis.yml

```diff
@@ -4,6 +4,7 @@ python:
   - 3.4
   - 3.5
   - 3.6
+  - 3.7
```

Comment:
3.7 needs `xenial` and `sudo`, see https://github.com/zopefoundation/zope.annotation/pull/17/files#diff-354f30a63fb0907d4ad57269548329e3R9 as an example